tutoroars/plugin.py

```diff
@@ -34,7 +34,7 @@
 
         # Make sure LMS / CMS have evnet-routing-backends installed
         # TODO: Do a new release and pin this! Also add config!
-        ("OPENEDX_EXTRA_PIP_REQUIREMENTS", "event-routing-backends"),
+        ("OPENEDX_EXTRA_PIP_REQUIREMENTS", ["edx-event-routing-backends"]),
```

Comment:
I should have noticed this issue when reviewing https://github.com/openedx/tutor-contrib-oars/pull/8, sorry!

Background:
1. The value needs to be an array, otherwise we get this indecipherable error during startup:

   ```
   Step 37/120 : RUN cd /openedx/requirements/   && touch ./private.txt   && pip install -r ./private.txt
    ---> Using cache
    ---> bdcf7bc3ac8d
   Step 38/120 : RUN pip install 'e'
    ---> Using cache
    ---> 36d41dafa618
   Step 39/120 : RUN pip install 'v'
    ---> Using cache
    ---> 556868ebcff8
   Step 40/120 : RUN pip install 'e'
    ---> Using cache
    ---> 908511ba6b1b
   Step 41/120 : RUN pip install 'n'
    ---> Running in 547e75ca229c
   ERROR: Could not find a version that satisfies the requirement n (from versions: none)
   ERROR: No matching distribution found for n
   ```
2. The [full pip package name is `edx-event-routing-backends`](https://pypi.org/project/edx-event-routing-backends/), so the full line needs to be:

   ```
   ("OPENEDX_EXTRA_PIP_REQUIREMENTS", ["edx-event-routing-backends"]),
   ```

Reply:
Seems like it was just a thing that somehow worked on my end, but we did update the Tutor docs to reflect this.